apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/SidePanelEditor.utils.tsx

```diff
@@ -740,12 +742,8 @@ export const updateTable = async ({
 
   await Promise.all([
     queryClient.invalidateQueries(sqlKeys.query(projectRef, ['foreign-key-constraints'])),
-    // invalidate list of tables, as well as visible individual tables
-    queryClient.invalidateQueries(['projects', projectRef, 'tables']),
```

Comment:
just to double check - we're removing invalidating projects here?

Reply:
I couldn't see this query key in use anymore